images/backups/Makefile

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 IMAGE_NAME = backups
-include ../../shared/Makefile.simple-image
+include ../../pkg/Makefile.simple-image
```

Comment:
should this file be in pkg?

Reply:
I can move it to `images/`, do you think it would be a better place?

Reply:
I think that's more reasonable than in pkg

Reply:
Agree, moved.